app/services/idv/steps/send_link_step.rb

```diff
@@ -54,6 +54,10 @@ def formatted_destination_phone
 
       def update_document_capture_session_requested_at(session_uuid)
         document_capture_session = DocumentCaptureSession.find_by(uuid: session_uuid)
+        if document_capture_session.cancelled_at
+          # See note at CancellationsController#cancel_document_capture_session
+          flow_session.delete(Idv::Steps::LinkSentStep.name)
```

Comment:
Could we have used the method for this?

```suggestion
          mark_step_incomplete(:link_sent)
```

Reply:
Also, I wonder if there would be any downside to just always marking `link_sent` incomplete when visiting this page, and not concerning ourselves with the `cancelled_at` consideration.

Reply:
I didn't realize we had that method 🤦 will make a PR to simplify.

Reply:
https://github.com/18F/identity-idp/pull/5772